terraform/modules/hub/modules/ecs_app/ecs.tf

```diff
@@ -8,12 +8,14 @@ module "cluster_ecs_roles" {
   deployment       = "${var.deployment}"
   service_name     = "${var.cluster}"
   tools_account_id = "${var.tools_account_id}"
+  image_name       = "${var.image_name}"
 }
 
 resource "aws_ecs_task_definition" "cluster" {
   family                = "${local.identifier}"
   container_definitions = "${var.task_definition}"
-  network_mode          = "awsvpc"
+  execution_role_arn    = "${module.cluster_ecs_roles.execution_role_arn}"
+  network_mode          = "bridge"
```

Comment:
this will lock us from fargate ecs.

Reply:
There are no plans of us moving to fargate, so it's ok.